src/jquery.ime.js

```diff
@@ -48,12 +48,20 @@
 		// This needs to be delayed here since extending language list happens at DOM ready
 		$.ime.defaults.languages = arrayKeys( $.ime.languages );
 		this.options = $.extend( {}, $.ime.defaults, options );
+		if ( this.options.imePath && this.options.imePath !== $.ime.path ) {
+			// Set the global IME path from the one specified to the instance
+			// TODO: remove this functionality and force clients to set the global
+			// IME path
+			$.ime.path = this.options.imePath;
+		}
```

Comment:
Basically we are giving priority for $.options.imePath if defined, but consistantly use $.ime.path in rest of the code. I think in that case the comparison `this.options.imePath !== $.ime.path`  is redundant right?